governance/proposals/012-set-protocol-fee-cross-bridge.js

```diff
@@ -0,0 +1,161 @@
+/**
+ * This proposal send calls accross the bridge to enable protocol fees
+ * on all supported networks.
+ */
+const { ADDRESS_ZERO, getNetwork } = require('@unlock-protocol/hardhat-helpers')
+const { Unlock } = require('@unlock-protocol/contracts')
+const { IConnext, targetChains } = require('../helpers/bridge')
+const { ethers } = require('hardhat')
+const PROTOCOL_FEE_IN_BASIS_POINTS = '100' // 1% in basis points
+
+const parseSetProtocolFeeCalls = async (destChainId) => {
+  const { unlockAddress } = await getNetwork(destChainId)
+
+  console.log(`Proposol to set protocolFee to ${ethers.formatEther(protocolFee)}
+    - unlock : ${unlockAddress}`)
+
+  // get Unlock interface
+  const { interface: unlockInterface } = await ethers.getContractAt(
+    Unlock.abi,
+    unlockAddress
+  )
+
+  const calldata = unlockInterface.encodeFunctionData('setProtocolFee', [
+    PROTOCOL_FEE_IN_BASIS_POINTS,
+  ])
+
+  // encode instructions to be executed by the SAFE
+  const moduleData = await ethers.defaultAbiCoder.encode(
+    ['address', 'uint256', 'bytes', 'bool'],
+    [
+      unlockAddress, // to
+      0, // value
+      calldata, // data
+      1, // operation: 0 for CALL, 1 for DELEGATECALL
+    ]
+  )
+
+  const oldProtocolFee = await unlockInterface.protocolFee()
```

Comment:
That won't work as you need a provider and the contract to actually get the value. The old value was null as the protocol fee has never been switched on so we can skip that step ?